prov/shm/src/smr_dsa.c

```diff
@@ -785,7 +785,7 @@ size_t smr_dsa_copy_from_sar(struct smr_ep *ep, struct smr_freestack *sar_pool,
 
 	assert(smr_env.use_dsa_sar);
 
-	if (resp->status != SMR_STATUS_SAR_READY)
+	if (resp->status != SMR_STATUS_SAR_BUFFER_FULL)
```

Comment:
This file uses different names than what's below.  Does this compile?

Reply:
No it does not 
```
 prov/shm/src/smr_dsa.c: In function ‘dsa_update_tx_entry’:
 prov/shm/src/smr_dsa.c:551:4: error: ‘SMR_STATUS_SAR_BUFFER_FULL’ undeclared (first use in this function); did you mean ‘SMR_STATUS_SAR_FULL’?
     SMR_STATUS_SAR_BUFFER_FULL : SMR_STATUS_SAR_BUFFER_EMPTY);
     ^~~~~~~~~~~~~~~~~~~~~~~~~~
     SMR_STATUS_SAR_FULL
 prov/shm/src/smr_dsa.c:551:4: note: each undeclared identifier is reported only once for each function it appears in
 prov/shm/src/smr_dsa.c:551:33: error: ‘SMR_STATUS_SAR_BUFFER_EMPTY’ undeclared (first use in this function); did you mean ‘SMR_STATUS_SAR_EMPTY’?
     SMR_STATUS_SAR_BUFFER_FULL : SMR_STATUS_SAR_BUFFER_EMPTY);
                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~
                                  SMR_STATUS_SAR_EMPTY
 prov/shm/src/smr_dsa.c:539:21: warning: variable ‘peer_smr’ set but not used [-Wunused-but-set-variable]
   struct smr_region *peer_smr;
                      ^~~~~~~~
 prov/shm/src/smr_dsa.c: In function ‘dsa_update_sar_entry’:
 prov/shm/src/smr_dsa.c:569:4: error: ‘SMR_STATUS_SAR_BUFFER_FULL’ undeclared (first use in this function); did you mean ‘SMR_STATUS_SAR_FULL’?
     SMR_STATUS_SAR_BUFFER_FULL : SMR_STATUS_SAR_BUFFER_EMPTY);
     ^~~~~~~~~~~~~~~~~~~~~~~~~~
     SMR_STATUS_SAR_FULL
 prov/shm/src/smr_dsa.c:569:33: error: ‘SMR_STATUS_SAR_BUFFER_EMPTY’ undeclared (first use in this function); did you mean ‘SMR_STATUS_SAR_EMPTY’?
     SMR_STATUS_SAR_BUFFER_FULL : SMR_STATUS_SAR_BUFFER_EMPTY);
                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~
                                  SMR_STATUS_SAR_EMPTY
 prov/shm/src/smr_dsa.c: In function ‘smr_dsa_copy_to_sar’:
 prov/shm/src/smr_dsa.c:764:22: error: ‘SMR_STATUS_SAR_BUFFER_EMPTY’ undeclared (first use in this function); did you mean ‘SMR_STATUS_SAR_EMPTY’?
   if (resp->status != SMR_STATUS_SAR_BUFFER_EMPTY)
                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~
                       SMR_STATUS_SAR_EMPTY
 prov/shm/src/smr_dsa.c: In function ‘smr_dsa_copy_from_sar’:
 prov/shm/src/smr_dsa.c:788:22: error: ‘SMR_STATUS_SAR_BUFFER_FULL’ undeclared (first use in this function); did you mean ‘SMR_STATUS_SAR_FULL’?
   if (resp->status != SMR_STATUS_SAR_BUFFER_FULL)
                       ^~~~~~~~~~~~~~~~~~~~~~~~~~
                       SMR_STATUS_SAR_FULL
   CC       prov/tcp/src/src_libfabric_la-xnet_domain.lo
```

Reply:
Sorry... my vscode symbol rename must have gotten messed up... fixed, and verified with grep that there are no more old naming.